packages/worker/src/api/controllers/global/users.ts

```diff
@@ -84,12 +142,67 @@ export const adminUser = async (ctx: any) => {
 
 export const destroy = async (ctx: any) => {
   const id = ctx.params.id
+  const db = tenancy.getGlobalDB()
```

Comment:
The changes here could be moved into the implementation of `users.destroy` so that we don't have split logic. 
Or based on the below comment about using pro for the group modifications it could read like
```ts
await users.destroy(id, ctx.user)
await groups.removeUser(groups, userId)
```
I would vote for putting the group updates inside `users.destroy` so that it's always captured.